metricbeat/module/mongodb/mongodb.go

```diff
@@ -123,7 +123,7 @@ func ParseURL(module mb.Module, host string) (mb.HostData, error) {
 		Username:                c.Username,
 		Password:                c.Password,
 	}
-	clientOptions.SetDirect(true)
```

Comment:
What if a direct connection is to be made? Does it work?

Reply:
This is possible through URI, as mentioned in the doc:

- module: mongodb
  hosts: ["mongodb://localhost:27017/?directConnection=true"]